### DISCUSSION TITLE
Mapping function of server side object is not working

### DISCUSSION THREAD

**Discussion by ostermann-felix** [neuefische/questions#175] [Web] [answered]
Hi friends, 

I have a problem with the map function I want to use to map the object that I get out of the database fetch.

It says (error - pages/content/LaienReanimation.js (29:17) @ LaienReanimation
error - TypeError: Cannot read properties of undefined (reading 'map'))
But nothing more...

![Bildschirmfoto 2022-09-27 um 14 29 55](https://user-images.githubusercontent.com/94837451/192528005-5e8e5423-c322-49f0-a5fb-5cf7c2e1e0ae.png)
![Bildschirmfoto 2022-09-27 um 14 37 08](https://user-images.githubusercontent.com/94837451/192528239-bd9fa633-bbe9-43b4-8dc3-e450a271088b.png)

**Comment by ThomasKirst** [comment]
How do you pass the `lessons` into your component?
Can you show us the part of the code which gets the lessons and the part where the prop is passed down (from parent component or page)?

**Reply by ostermann-felix** [reply]
![Bildschirmfoto 2022-09-27 um 14 45 49](https://user-images.githubusercontent.com/94837451/192530046-771338e9-1694-479e-b50a-3756ebd01802.png)

**Comment by ThomasKirst** [comment, marked as answer]
`props`, not `probs` please ;)
Same for `getServerSideProps()`

**Reply by ostermann-felix** [reply]
Now this is the following error. But I checked and my dbConnect is a function 

![Bildschirmfoto 2022-09-27 um 14 53 45](https://user-images.githubusercontent.com/94837451/192531812-4c768ba2-0372-4318-9325-caed0782bef2.png)

**Reply by NikRoe** [reply]
Can you send us a link to your Pull Request? :)

**Reply by ostermann-felix** [reply]
https://github.com/ostermann-felix/capstone-project/pull/9

**Reply by NikRoe** [reply]
Hey Felix,
you need to export your function `dbConnect()`. Otherwise you cannot import it in another file.

And I would advise moving the file to the root of your project.

Hope this already helps 🤞

**Reply by ostermann-felix** [reply]
Helped, but now the link doesn't work any more and this is displayed at the console

![Bildschirmfoto 2022-09-27 um 15 54 22](https://user-images.githubusercontent.com/94837451/192546092-2de35a78-e769-48a1-a777-4df1fe35817c.png)

**Reply by NikRoe** [reply]
marking this as answered, since we could solve it in person
(Password in `.env.local` had a spelling mistake and keys in db started with capital letter while they were trying to be used starting with small letters)
